ray-operator/config/samples/vllm/ray-service.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: ray.io/v1
+kind: RayService
+metadata:
+  name: llama-3-8b
+spec:
+  serveConfigV2: |
+    applications:
+    - name: llm
+      route_prefix: /
+      import_path:  ray-operator.config.samples.vllm.serve:model
+      deployments:
+      - name: VLLMDeployment
+        num_replicas: 1
+        ray_actor_options:
+          num_cpus: 8
+          # NOTE: num_gpus is set automatically based on TENSOR_PARALLELISM
+      runtime_env:
+        working_dir: "https://github.com/ray-project/kuberay/archive/master.zip"
+        pip: ["vllm==0.4.3"]
```

Comment:
how about using the latest one v0.5.4 https://github.com/vllm-project/vllm/releases?

Reply:
Let me test it with v0.5.4

Reply:
Tested and updated to use v0.5.4, it required a few changes from the example in https://docs.ray.io/en/latest/serve/tutorials/vllm-example.html though